.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+If your issue is not following the template, the maintainer may close the issue without notifications.
+
+
+Issue Type: New Feature Request
+--------------------------------
+
+* Write your concrete practical scenarios with the new feature you want.
+* Write your expectation how your suggestion helps others as well.
+* Do not file a request like "support a new input/output/filter" here.
+  * That's not for here.
+  * Embulk is plugin-based. Such inputs/outputs/filters are to be implemented as plugins, not in the core.
+  * It is recommended for you to search for existing plugins at first.
```

Comment:
What do you think to add this URL here http://www.embulk.org/plugins/ ?

Reply:
Thanks. Sounds like a good idea. :)